nano/node/nodeconfig.cpp

```diff
@@ -137,7 +137,7 @@ nano::error nano::node_config::serialize_toml (nano::tomlconfig & toml) const
 		work_peers_l->push_back (boost::str (boost::format ("%1%:%2%") % i->first % i->second));
 	}
 
-	auto preconfigured_peers_l (toml.create_array ("preconfigured_peers", "A list of \"address\" (hostname or ipv6 notation ip address) entries to identify preconfigured peers."));
```

Comment:
I do not think that that statement is correct.
NANO_DEFAULT_PEER overrides the peering address.
I think, but I am not sure, that the contents of NANO_DEFAULT_PEER are added to preconfigured_peers.

Reply:
Oh you're right, `emplace_back`. Let me fix that